### DISCUSSION TITLE
Can I hide the 'Sign in' button?

### DISCUSSION THREAD

**Discussion by sdemura** [zed-industries/zed#11054] [Feature Requests]
I have absolutely zero desire to connect Zed to Github (most of my work is done with Gitlab). Is there a way to hide it?

<img width="109" alt="image" src="https://github.com/zed-industries/zed/assets/11758459/d561934f-f8a8-4a27-8931-b6dc9b4a704a">

**Comment by tednaaa** [comment]
seems no, there is no setting for that, just wait for developers, maybe they will create for gitlab and etc later

**Reply by sdemura** [reply]
hopefully just an option to disable it altogether :)

**Comment by canadaduane** [comment]
I'd like this as well. Speaking for myself: A primary, must-be-visible "Sign in" link vibes a proprietary product that puts company first. A softer touch here would be a trust-building step.

**Comment by evanheller** [comment]
The developers appear quite set on not giving users this option. There is some discussion of this in issue #12325; as well as in #11802.

**Comment by dzierzanowski** [comment]
This does not bode well for the future of this project. In the past, open-source tools that had this indicator were later rug-pulled and paywalled.

**Comment by ghost** [comment]
`  "title_bar": {
    "show_sign_in": false
  }`

https://zed.dev/docs/visual-customization#titlebar

**Reply by canadaduane** [reply]
Thank you for pointing this out! This works beautifully.

**Reply by ElnuDev** [reply]
Thank you! This doesn't seem to remove the little "disconnected" crossed out cloud, though.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
